ArduCopter/mode_plancktracking.cpp

```diff
@@ -37,20 +37,38 @@ void ModePlanckTracking::run() {
 
     // check if gimbal steering needs to controlling the vehicle yaw
     AP_Mount *mount = AP::mount();
-    if ((mount != nullptr) && (mount->mount_yaw_follow_mode == AP_Mount::vehicle_yaw_follows_gimbal)) {
-        float angle_deg = mount->get_follow_yaw_rate();
-        int8_t direction = 1;
-        bool relative_angle = true;
-        if (angle_deg < 0.0f) {
-            angle_deg = -angle_deg;
-            direction = -1;
+    bool paylod_yaw_rate = false;
```

Comment:
Don't these changes need to be made to `poshold` as well (or any other location the gimbal yaw commands are used)?

Reply:
Ah, yes, at least part of that should be there as well. I had assumed poshold would never be used, but we should have it there just in case. I will make the change.